### PR TITLE
#2307 Add a 'no options' message when there are no Advanced Options to present for a brick

### LINKED ISSUE #2307: Blank advanced options card showing in v3
Since we removed the `templateEngine` field in v3, there are some bricks that have a blank advanced options section (see screenshot, which is from the Document Builder):

image

We should show a `text-muted` message here that says no Advance Options available? That's probably preferable to hiding the section

### PR DESCRIPTION
**Pull request by BLoe**
Closes #2307 

Adds this message:

image

### CODE REVIEW

**Comment by fregante** [comment]
It looks like the new snapshots are missing

File: src/devTools/editor/tabs/effect/v3/BlockConfiguration.tsx [hunk 1]
```diff
@@ -91,6 +91,23 @@ const BlockConfiguration: React.FunctionComponent<{
...
+  );
+
+  const showIfAndTarget = useMemo(() => blockType && blockType !== "renderer", [
+    blockType,
+  ]);
+
+  const noAdvancedOptions = useMemo(() => !showRootMode && !showIfAndTarget, [
+    showIfAndTarget,
+    showRootMode,
+  ]);
```
Comment (BALEHOK) [inline src/devTools/editor/tabs/effect/v3/BlockConfiguration.tsx:109]: These are super simple boolean operations, no reason for `useMemo` here. In general I do not favor occasional performance optimizations, when there's no proven need for caching, don't cache.
Keep in mind that `useMemo` (and `useCallback`) add overhead for caching.
Here is a benchmark for `useMemo` vs calculations on every render https://medium.com/swlh/should-you-use-usememo-in-react-a-benchmarked-analysis-159faf6609b7

The bottom line is with a number of loops < 100, `useMemo` hardly seems interesting.

Comment (BALEHOK) [reply src/devTools/editor/tabs/effect/v3/BlockConfiguration.tsx:109]: Actually, after reading the article I feel stronger about not having `useMemo` for `showRootMode` as well.

**Review by BALEHOK — CHANGES_REQUESTED** [review]

File: src/devTools/editor/tabs/effect/v3/BlockConfiguration.tsx (hunk 1 above)
Comment (BLoe) [reply src/devTools/editor/tabs/effect/v3/BlockConfiguration.tsx]: Done, removed the memos
